graph/elements/node.py

```diff
@@ -15,5 +15,5 @@ def __init__(self, x: float, y: float, is_hanging: bool, label: str = "V"):
         self.label = label
 
     def __str__(self):
-        string = f"Node: {self.x} {self.y}, {self.is_hanging} ({self.label})"
+        string = f"Node: {self.x}.{self.y}, {self.is_hanging} ({self.label})"
```

Comment:
I removed it deliberately, debugging with this dot is fucked up, you see coords like this `0.0.1.0`, it took me 10 minutes to understand what happens there